### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by t0yamont**
Header Layout: For the header layout I changed to a felx container with justify-between and text-center classes, making the items more eavenly distributed on the line with centered-align text.

Logo: I made the logo's height and horizontal margins dynamic, depending on if the drawer is open or not.

Navigation Links: I changed the navigation links from vertical to a horizontal layout and made them expand on the navigation bar without taking actual page space. Also added additional classes for transformation and hover effect.

### CODE REVIEW

File: components/homepage/Hero.vue
Comment (g30r93g) [inline components/homepage/Hero.vue:1]: requires conversion to tailwind

**Review by g30r93g — CHANGES_REQUESTED** [review]
Convert transition to tailwind. Use Google Gemini or ChatGPT for assistance

**Comment by t0yamont** [comment]
CSS changed to Tailwind.

**Review by g30r93g — CHANGES_REQUESTED** [review]
Please rerun with `yarn` instead of `npm` and ensure new `yarn.lock` is generated

**Comment by t0yamont** [comment]
Deleted the package-lock.json file and installed gsap dependencie through yarn

**Pull request closed without merging**